modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2TeamMembersListing.testcase

```diff
@@ -0,0 +1,230 @@
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "${testSiteName}");
+		}
+	}
+
+	@description = "Automation ID: LPS-149746 | Test Summary: Verify if the team member table contains the following data (Name - Email - Support Seat - Role - Status)"
+	@priority = "4"
+	test AssertAllItemsInTheTable {
+		property portal.upstream = "quarantine";
+
+		// "Reason: Not eble to reproduce scenario without OKTA workaround."
+
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account DXP");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		}
+
+		task ("Login in Okta") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			CP2TeamMembers.teamMembersFlow();
+		}
+
+		task ("Assert the details of the table") {
+			CP2TeamMembers.viewTableValue(
+				email = "test@liferay.com",
+				name = "Test Test",
+				role = "Administrator",
+				status = "Active");
+		}
+
+		task ("Delete the entries and objects created") {
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-002");
+		}
+	}
+
+	@description = "Automation ID: LPS-149752 | Test Summary: Verify that in the table the guest user is below the current user and its status is displayed as invited"
+	@priority = "5"
+	test AssertTheStatusAndOrderOfTheGuestUser {
+
+		// TODO LPS-149752 archive pending implementation
+
+	}
+
+	@description = "Automation ID: LPS-149754 | Test Summary: Verify that the text in the upper right corner is similar to 'Support Seats: 1 of 3 available', if the project has a support seats limit"
+	@priority = "4"
+	test CanViewTextIfProjectHasLimitOfSupportSeats {
+		property portal.upstream = "quarantine";
+
+		// "Reason: Not eble to reproduce scenario without OKTA workaround."
+
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account DXP");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		}
+
+		task ("Login in Okta") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			CP2TeamMembers.teamMembersFlow();
+		}
+
+		task ("Assert in the number of support seats available") {
+			CP2TeamMembers.viewSupportSeatsAvailable(available = "2 of 3 available");
+		}
```

Comment:
@luanakarla since we are only adding this validation to this task, do you think this assert could be added to AssertAllItemsInTheTable test? It looks like we are performing the same steps. We could combine both tests into only 1 and decrease the use of CI servers.

Reply:
Done.